scripts/run_integration_tests_aarch64.sh

```diff
@@ -209,6 +209,28 @@ TARGET_CC="musl-gcc"
 CFLAGS="-I /usr/include/aarch64-linux-musl/ -idirafter /usr/include/"
 fi
 
+# Use device mapper to create a snapshot of the Ubuntu Bionic image
+bionic_img_blk_size=$(du -b -B 512 /root/workloads/bionic-server-cloudimg-arm64.raw | awk '{print $1;}')
+bionic_loop_device=$(losetup --find --show --read-only /root/workloads/bionic-server-cloudimg-arm64.raw)
+dmsetup create bionic-base --table "0 $bionic_img_blk_size linear $bionic_loop_device 0"
+dmsetup mknodes
+dmsetup create bionic-snapshot-base --table "0 $bionic_img_blk_size snapshot-origin /dev/mapper/bionic-base"
+dmsetup mknodes
+# Use device mapper to create a snapshot of the Ubuntu Focal image
+focal_img_blk_size=$(du -b -B 512 /root/workloads/focal-server-cloudimg-arm64-custom.raw | awk '{print $1;}')
+focal_loop_device=$(losetup --find --show --read-only /root/workloads/focal-server-cloudimg-arm64-custom.raw)
+dmsetup create focal-base --table "0 $focal_img_blk_size linear $focal_loop_device 0"
+dmsetup mknodes
+dmsetup create focal-snapshot-base --table "0 $focal_img_blk_size snapshot-origin /dev/mapper/focal-base"
+dmsetup mknodes
+# Use device mapper to create a snapshot of the Ubuntu Focal QCOW image
+focal_qcow_img_blk_size=$(du -b -B 512 /root/workloads/focal-server-cloudimg-arm64-custom.qcow2 | awk '{print $1;}')
```

Comment:
We never use this so why not get rid of it (in fact it won't work.)

Reply:
Fixed!